appinfo.go

```diff
@@ -1,12 +1,22 @@
 package appwrap
 
-import "google.golang.org/appengine"
+import (
+	"context"
+	"fmt"
+	"os"
+	"sync"
+
+	"cloud.google.com/go/compute/metadata"
+	"google.golang.org/appengine"
+	istio "istio.io/client-go/pkg/clientset/versioned"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
 
 type AppengineInfo interface {
 	DataProjectID() string
 	NativeProjectID() string
 	InstanceID() string
-	ModuleDefaultVersionID(moduleName string) (string, error)
```

Comment:
are we not using this anywhere?

Reply:
Nope

Reply:
We could just leave it though if we wanted to be careful - maybe a project unknown to us uses it. Maggie removed this in her POC PR, probably so we don't have to implement it in the k8s implementation. I double checked in pendo-appengine, and didnt find usage. 

Opinions?